plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -9,6 +13,9 @@
   private String eclipseProjectName;
   private String packageName;
   private String eclipseProjectDirectory;
+  private URI eclipseProjectLocationUri;
+  private IProject project;
+  private boolean useDefaultLocation;
```

Comment:
Does `useDefaultLocation` belong here?  `useDefaultLocation` seems like UI-level state, used to decide what's put into `eclipseProjectDirectory`, `eclipseProjectLocationUri`, and `project`.